stestr/scheduler.py

```diff
@@ -108,7 +108,7 @@ def consume_queue(groups):
             for partition in partitions:
                 temp_part = list(partition)
                 random.shuffle(temp_part)
-                out_parts.append(set(temp_part))
+                out_parts.append(list(temp_part))
```

Comment:
So why the change in type here? I was casting this with set() because that's what normally gets returned outside the randomize case.